test/config.json

```diff
@@ -3,6 +3,6 @@
     "host": "127.0.0.1",
     "port": 8983,
     "path": "/solr",
-    "core": null
+    "core": "testcore"
```

Comment:
It was failing on a newer Solr with 404 error, but worked after I added explicit core. Since this doesn't affect lib implementation per se, this isn't a breaking change.